contrib/localnet/orchestrator/smoketest/test_stress.go

```diff
@@ -158,13 +159,24 @@ func StressTest(_ *cobra.Command, _ []string) {
 	)
 
 	// If stress test is running on local docker environment
-	if stressTestArgs.local {
+	if stressTestArgs.network == "PRIVNET" {
 		smokeTest.TestSetupZetaTokenAndConnectorAndZEVMContracts()
 		smokeTest.TestDepositEtherIntoZRC20()
 		smokeTest.TestSendZetaIn()
+	} else {
+		ethZRC20Addr, _ := smokeTest.SystemContract.GasCoinZRC20ByChainId(&bind.CallOpts{}, big.NewInt(5))
+		smokeTest.ETHZRC20Addr = ethZRC20Addr
+		smokeTest.ETHZRC20, _ = zrc20.NewZRC20(smokeTest.ETHZRC20Addr, smokeTest.zevmClient)
```

Comment:
As far as I can tell, we support either local or testnet?

I would suggest checking here `stressTestArgs.network == "TESTNET"` and returning error otherwise